README.md

```diff
@@ -77,7 +77,7 @@ puts decoded_token
 * HS512 - HMAC using SHA-512 hash algorithm
 
 ```ruby
-# The secret must be a string. A JWT::DecodeError will be raised if it isn't provided.
+# The secret must be a string. With OpenSSL 3.0, JWT::DecodeError will be raised if it isn't provided.
```

Comment:
It's the combination of OpenSSL 3.0 and the openssl gem prior to `3.0.1`. So this comment will become less significant when time moves forward.